lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -140,12 +140,8 @@ def add_creditcard_for_tokenization(params, payment_method, options)
         params[:ta_token] = options[:ta_token]
         params[:callback] = 'Payeezy.callback'
```

Comment:
According to https://developer.payeezy.com/payeezy-api/apis/post/transactions/tokens we don't need to send in the `callback` and `js_security_key`.

Reply:
Also, there's a strong possibility we'll no longer need to send in the the `apikey` param in `add_creditcard_for_tokenization` anymore https://github.com/activemerchant/active_merchant/blob/ccca1e2e72c3277c1257e27c911acf0d63b95d66/lib/active_merchant/billing/gateways/payeezy.rb#L137

It's worth looking into this since we *should* already be performing the authentication Payeezy expects in https://github.com/activemerchant/active_merchant/blob/ccca1e2e72c3277c1257e27c911acf0d63b95d66/lib/active_merchant/billing/gateways/payeezy.rb#L301-L312

Reply:
@dtykocki thanks for the catch. Will ✂️ that.